app/helpers/application_helper.rb

```diff
@@ -65,11 +65,18 @@ def govuk_assets_path
   end
 
   def header_component_options(user:, can_manage_users:)
+    links = Hash.new({}).update( # rubocop:disable Rails/SaveBang
```

Comment:
It took me a moment to figure out how this worked (though I may just be slow after not enough sleep last night 😪 )...might be worth a chat to see if there is a clearer way?

Reply:
I would like a clearer way haha...

Reply:
I've refactored to make more explicit, thanks for the nudge